pkg/runner/runner_test.go

```diff
@@ -20,11 +20,9 @@ var tests = []struct {
 		"missing plugin.json",
 		"missing module.js",
 		"missing README.md",
-		"unsigned plugin",
```

Comment:
It is odd that these tests are different because of the manifest changes but it is related to dependencies and how the runner runs them. 

This PR https://github.com/grafana/plugin-validator/pull/147 is trying to fix these inconsistencies.